src/Provider/AbstractProvider.php

```diff
@@ -837,7 +837,7 @@ protected function prepareAccessTokenResponse(array $result)
      *
      * @param  array $response
      * @param  AbstractGrant $grant
-     * @return AccessTokenInterface
+     * @return AccessToken
```

Comment:
Since `AccessToken` implements `AccessTokenInterface`, I don't understand why Psalm would complain about this.

Is this still a problem in the latest version of Psalm?